src/main/xerus/monstercat/Settings.kt

```diff
@@ -42,6 +43,26 @@ object Settings : SettingsNode("xerus/monsterutilities") {
 	
 	val FILENAMEPATTERN = create("updatePattern", "MonsterUtilities-%version%.jar")
 	
+	enum class ConnectionSpeed(val maxConnections: Int, val display: String) : Named {
+		DIALUP(5, "Dial-up (100 kb/s)"),
+		ADSL(30, "ADSL / 3G (10 Mb/s)"),
+		CABLE(150, "Cable / 4G (100 Mb/s)"),
+		FIBER(300, "Fiber (300+ Mb/s)");
+
+		override val displayName: String
```

Comment:
no need to declare this variable separately. You can override it right in the constructor by simply renaming `display` to `displayName`.

Reply:
eaeaedfcf9a64f37ac1b0ddbdd2b60139641d54b

Reply:
eaeaedfcf9a64f37ac1b0ddbdd2b60139641d54b